tests/end_to_end/test_best_practices_content_pages.py

```diff
@@ -0,0 +1,72 @@
+import os
+import re
+
+from playwright.sync_api import expect
+
+from tests.end_to_end.conftest import check_axe_report
+
+E2E_TEST_URI = os.getenv("NOTIFY_E2E_TEST_URI")
+
+
+def test_best_practices_side_menu(authenticated_page):
+    page = authenticated_page
+
+    page.goto(f"{E2E_TEST_URI}/best-practices")
+
+    page.wait_for_load_state("domcontentloaded")
+    check_axe_report(page)
+
+    response = page.request.get(f"{E2E_TEST_URI}/test/feature-flags")
```

Comment:
Just double checking my understanding:  this is just getting the response of the feature-flags URI, not actually going to the page, right?

Reply:
That's correct, my understanding is that the `get` retrieves the response from the server without navigating to the page.

Reply:
Thank you!